assets/blocks/course-list-block/index.js

```diff
@@ -120,6 +121,31 @@ const hideUnnecessarySettingsForCourseList = () => {
 	} );
 };
 
+function addWrapperAroundFeaturedImageBlock( settings, name ) {
```

Comment:
This could also use a JSDoc comment.

Reply:
ff4fbfa4ec6cf59e228ea2edcce8d48ae648f073